Added a scene and scripts for the bouncy clouds

Shubbii opened the pull request.

Shubbii committed:
Added a scene and scripts for the bouncy clouds

natejank reviewed (commented):
Also you need to center everything to 0, 0 in bouncyclouds.tscn

rn they're a little off center

Great work though :^)

Commit:
Moved location and consolidated BouncyClouds.tscn

committing so I can pull from develop

Commit:
Merge branch 'develop' of https://github.com/8-bit-Falcons/June-2021-Game-Jam into bouncy-clouds

Commit:
Various fixes with script and scene; added tileset

simplified scene to only have 1 hitbox
 - allowed for only 1 script

used collision layers to detect players

used constants

centered sprites

removed test scene

added tileset & script for easy placement

had a good ol' time :^)

Commit:
Added utility class for using scenes in a tileset

Used said utility class in the code for bouncyclouds

modified the disappearing platform code to also use the utility class
